src/env.h

```diff
@@ -331,6 +329,8 @@ constexpr size_t kFsStatsBufferLength =
   V(next_string, "next")                                                       \
   V(nistcurve_string, "nistCurve")                                             \
   V(node_string, "node")                                                       \
+  V(nodejs_clone_string, "nodejs.clone")                                       \
```

Comment:
I think what @jasnell meant was that you can add the symbols to `ENVIRONMENT_STRONG_PERSISTENT_VALUES` instead and create the references to them in `InitializeMainContext`, as a shortcut in C++ and avoid calling `Symbol::For` all the time in `JSTransferrable` (the latter incurs a dictionary lookup in the per-isolate symbol table).

Reply:
Another alternative would be to create another list (`PER_ISOLATE_PUBLIC_SYMBOL_PROPERTIES` perhaps) in the isolate data that creates symbols using `Symbol::For` instead of `Symbol::New`, either way they are just for creating some shortcuts in C++.

Reply:
@joyeecheung The second option sounds better because we already have a similar `PER_ISOLATE_PRIVATE_SYMBOL_PROPERTIES` macro. Done in https://github.com/nodejs/node/pull/41148/commits/012d03bfe5bf122755af1446500ddf1fd54e27e1.